### ISSUE TITLE
Failing literal search

### ISSUE THREAD

**Issue by mikesname** [EHRI/ehri-search-tools#68]
Archives nationales d'outre-mer

Only works when quoted.

**Comment by mikesname** [comment]
@juntezhang thanks, I'll check that out! Good to hear from you, hope things are going well in NL.

**Comment by mikesname** [comment]
It turns out that this issue was a bit more straightforward than anticipated, since we already have fields set up with the elision filter for various languages. We just weren't using them in this search :frowning_man:. 

This probably wouldn't have been noticed, except we added a "mimimum must match" parameter more recently, which in combination the language analysis setting for "generic" text, caused this search to fail.

I have put a fix in place that makes use of the language-aware fields, in addition to some extra testing, and will review at a later date...

**Comment by juntezhang** [comment]
Good catch! Adding more tests is definitely a good idea. Are you currently
only working on this project?

**Comment by mikesname** [comment]
Yes, way past time we had a proper comprehensive set of search tests, but no-one's had the time yet. I am not alone on this project thankfully (:wave: @mdermentzi @herminiogg ), though I'm responsible for most of the technical debt in this particular area!
